neurokit2/ecg/ecg_findpeaks.py

```diff
@@ -131,34 +131,13 @@ def ecg_findpeaks(ecg_cleaned, sampling_rate=1000, method="neurokit", show=False
 
     method = method.lower()  # remove capitalised letters
     # Run peak detection algorithm
-    if method in ["nk", "nk2", "neurokit", "neurokit2"]:
-        rpeaks = _ecg_findpeaks_neurokit(ecg_cleaned, sampling_rate, show=show)
-    elif method in ["pantompkins", "pantompkins1985"]:
-        rpeaks = _ecg_findpeaks_pantompkins(ecg_cleaned, sampling_rate)
-    elif method in ["nabian", "nabian2018"]:
-        rpeaks = _ecg_findpeaks_nabian2018(ecg_cleaned, sampling_rate)
-    elif method in ["gamboa2008", "gamboa"]:
-        rpeaks = _ecg_findpeaks_gamboa(ecg_cleaned, sampling_rate)
-    elif method in ["ssf", "slopesumfunction", "zong", "zong2003"]:
-        rpeaks = _ecg_findpeaks_ssf(ecg_cleaned, sampling_rate)
-    elif method in ["hamilton", "hamilton2002"]:
-        rpeaks = _ecg_findpeaks_hamilton(ecg_cleaned, sampling_rate)
-    elif method in ["christov", "christov2004"]:
-        rpeaks = _ecg_findpeaks_christov(ecg_cleaned, sampling_rate)
-    elif method in ["engzee", "engzee2012", "engzeemod", "engzeemod2012"]:
-        rpeaks = _ecg_findpeaks_engzee(ecg_cleaned, sampling_rate)
-    elif method in ["elgendi", "elgendi2010"]:
-        rpeaks = _ecg_findpeaks_elgendi(ecg_cleaned, sampling_rate)
-    elif method in ["kalidas2017", "swt", "kalidas", "kalidastamil", "kalidastamil2017"]:
-        rpeaks = _ecg_findpeaks_kalidas(ecg_cleaned, sampling_rate)
-    elif method in ["martinez2003", "martinez"]:
-        rpeaks = _ecg_findpeaks_WT(ecg_cleaned, sampling_rate)
-    elif method in ["rodrigues2020", "rodrigues2021", "rodrigues", "asi"]:
-        rpeaks = _ecg_findpeaks_rodrigues(ecg_cleaned, sampling_rate)
-    elif method in ["promac", "all"]:
-        rpeaks = _ecg_findpeaks_promac(ecg_cleaned, sampling_rate=sampling_rate, threshold=0.33, show=show)
+    if method in all_methods.keys():
+        rpeaks = all_methods[method](ecg_cleaned, sampling_rate=sampling_rate, show=show, **kwargs)
```

Comment:
That's a good idea but I wonder if it makes it less verbose in the end since there is the big dictionary that takes even more space 🤔 are there any other advantages to this change?